zerver/tests/test_export.py

```diff
@@ -289,3 +294,10 @@ def find_by_id(table: str, db_id: int) -> Dict[str, Any]:
         dummy_user_emails = get_set('zerver_userprofile_mirrordummy', 'email')
         self.assertIn(self.example_email('iago'), dummy_user_emails)
         self.assertNotIn(self.example_email('cordelia'), dummy_user_emails)
+
+    @use_s3_backend
+    def test_import_files_from_s3(self):
```

Comment:
The mypy error in CI is because you need to add an `-> None` at the end of this line.